metadata-models/src/main/pegasus/com/linkedin/schema/EditableSchemaFieldInfo.pdl

```diff
@@ -1,60 +1,27 @@
 namespace com.linkedin.schema
-
-import com.linkedin.common.GlobalTags
-import com.linkedin.common.GlossaryTerms
+import com.linkedin.businessattribute.BusinessAttributeAssociation
 
 /**
  * SchemaField to describe metadata related to dataset schema.
  */
-record EditableSchemaFieldInfo {
-  /**
-   * FieldPath uniquely identifying the SchemaField this metadata is associated with
-   */
-  fieldPath: string
-
-  /**
-   * Description
-   */
-  @Searchable = {
-    "fieldName": "editedFieldDescriptions",
-    "fieldType": "TEXT",
-    "boostScore": 0.1
-  }
-  description: optional string
-
-  /**
-   * Tags associated with the field
-   */
-  @Relationship = {
-    "/tags/*/tag": {
-      "name": "EditableSchemaFieldTaggedWith",
-        "entityTypes": [ "tag" ]
-    }
-  }
-  @Searchable = {
-    "/tags/*/tag": {
-      "fieldName": "editedFieldTags",
-      "fieldType": "URN",
-      "boostScore": 0.5
-    }
-  }
-  globalTags: optional GlobalTags
+record EditableSchemaFieldInfo includes EditableSchemaFieldBase {
 
-  /**
-   * Glossary terms associated with the field
-   */
-   @Relationship = {
-      "/terms/*/urn": {
-        "name": "EditableSchemaFieldWithGlossaryTerm",
-        "entityTypes": [ "glossaryTerm" ]
-     }
-   }
-   @Searchable = {
-    "/terms/*/urn": {
-      "fieldName": "editedFieldGlossaryTerms",
-      "fieldType": "URN",
-      "boostScore": 0.5
-    }
-  }
-  glossaryTerms: optional GlossaryTerms
+      /**
+       * Business Attribute for this field.
+       */
+      @Relationship = {
+        "/destinationUrn": {
+          "name": "EditableSchemaFieldWithBusinessAttribute",
+          "entityTypes": [ "businessAttribute" ]
+        }
+      }
+      @SearchableRef = {
+          "/destinationUrn": {
+            "fieldName": "editedFieldBusinessAttributeRef",
+            "fieldType": "URN",
+            "boostScore": 0.5,
+            "refType": "businessAttribute"
+          }
+      }
+    businessAttribute: optional BusinessAttributeAssociation
```

Comment:
When tags and terms were originally added to schema fields, schema fields were not a top-level entity. 
At this point - we have that support. 
What happens if we attach the businessAttribute aspect directly to the schemaField? Similar to how we attached structuredProperties.

Reply:
This development has started before structured properties were introduced, so we have modelled it similar to how tags and glossary terms were being modelled in editableSchemaFiledInfo record. We will analyse the approach you suggested

Reply:
@shirshanka as suggested, business attribute now lives in schema field entity